src/components/UI/Sidebar/index.css

```diff
@@ -60,3 +68,104 @@
   justify-content: center;
   color: var(--accent);
 }
+
+.currentDownloads {
+  grid-area: utils;
+  align-self: end;
+  margin: 0 4px var(--sidebar-vertical-padding)
+    var(--sidebar-horizontal-padding);
+}
+
+.divider {
+  border: solid 1px var(--divider);
+  opacity: 0.2;
+  margin: 4px 8px;
+}
+
+.heroicVersion {
+  grid-area: version;
+  margin: 0 4px var(--sidebar-vertical-padding)
+    var(--sidebar-horizontal-padding);
+}
+
+.collapseIcon {
+  position: fixed;
+  left: calc(var(--sidebar-width) - 8px);
+  bottom: 55vh;
+  align-items: center;
+  cursor: pointer;
+  opacity: 0;
+  transition: 500ms;
```

Comment:
definitely my idea about the icon fadeout delay thing was a bad idea xD

Reply:
But do you we should remove it?

Reply:
after playing a bit with the CSS, I think we can use this:

change that line to be `transition: opacity 500ms 500ms;` so we only transition the opacity and not the position, with a small delay

then replace all the other styles for the transition:

```css
.Sidebar:hover .collapseIcon {
  transition: opacity 500ms;
  opacity: 1;
}

.Sidebar.collapsed:hover .collapseIcon {
  transition: opacity 3s;
  opacity: 1s;
}

.Sidebar.collapsed .collapseIcon {
  transition: opacity 1s;
}
```

with only this one:

```css
.Sidebar:hover .collapseIcon {
  opacity: 1;
}
```

with those changes there's a delay, then it fades in and it follows the sidebar change immediately when expanded/collapsed

Reply:
done! 
Thanks for the help 👍🏽